app/javascript/mastodon/reducers/missed_updates.ts

```diff
@@ -0,0 +1,32 @@
+import { Map } from 'immutable';
+import type { Action } from 'redux';
+import { NOTIFICATIONS_UPDATE } from '../actions/notifications';
+import type { MastodonMap } from '../../types/util';
+import { focusApp, unfocusApp } from '../actions/app';
+
+type MissedUpdatesState = MastodonMap<{
```

Comment:
You should be able to use `Map` from immutable here

Reply:
Type difinition for Map of Immutable are `Map<K, V>`.
This means that `state.get('focused')` will infer to `boolean | number`.

so, I replace Map by Record.

Reply:
Yes this seems more correct than the previous behaviour.

I worry a bit that changing the JS object from `ImmutableMap` to `Record` can break things, but this state is not used a lot and I dont think it will have an impact here.